### PR TITLE
Add more checks

### PR DESCRIPTION
**Pull request by marians**
Here I'm pushing more of the current state of the acceptance testing.

### Preview

image

### CODE REVIEW

File: helloworld-manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  type: ClusterIP
+  ports:
+  - port: 8080
+  selector:
+    app: helloworld
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: helloworld
```
Comment (xh3b4sd) [inline helloworld-manifest.yaml]: Why don't we use https://github.com/giantswarm/e2e-app as we do in our e2e tests?

File: kubeconfig_uat_zwm5s_60d5f81ac6c5fc15c236362b677b1a4c18f9b06e.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
...N0VmTHplMHpJZVltMmNxaTZvMitreTdYNktodk1VZUZiRDVQRgpUUDhDNzZ3WHQ2R1BOQWNYTlN3dEp3aHBPdC9jV3BDVFFCVlVjREdXRHZiTEFvZHN1cHI1eUNLcHFVVHhLeklRCjVtSkhEUlQ4K3VlL1VMYjFNTFVZV2R2d252QVlDQXRSQ2V6ZURZdGZPYkZqclgxYVlEV2ovV0VDZ1lFQXdaZ1oKOU9pdVRXYzBGNmV3cjJ3dFlhRjB5alVqdTZXTzZVNXc5R1FLcjdYTU43eWVkazlaYVhKdUFzTGpVMDg0RGk0bApnR1ZpRnFCL1NKc3U4TDNCbnhsc3l6OGw3Y0dDb1BjQ3ZhYzJucVRHc2c1NG53MDZQM3ZEaS94Q2ZVTVc5MXZqCjc3b01ERFpBRHNXV21YRWlXbFVtMUVhVUpVbnZOUU00d0prOU4wTUNnWUJvZE9FQ0dYZ2F0N0EvMWV6ekY5dDMKS1huOE5OL2lpeklKWDNOWE80Yi9kVElEcW5FTmM1U3dkbGR1YUw3WTZGU08zTTZtN2RDQU5mMjdjNE9oZC94cQo3YlNsaXVNdE52UWl2eU5iYlRJZlNsUTdnZnowQTA4Q0NLS3dFc1grVHY3OXo5ZU94bVJJUVpweDFJR1RUOHVtCmFrVDVLS3lKTlNlWnIyUzFSTUFRQVFLQmdBdVVYWGtZc2RSUFovNUNPWDlFcVRJOXhnNlNQeURxTmpJZVR3TlAKT0pScCsxMGFHeUJZZnJDZjk1U1NzNlk0QnRKU1VFdnR4SUExK1djT1lUQmxLVCtxOWFieWtBSTRvT2VmM3JvWgpWbHNHNEJ1bVlmQ2hpZXBpTE1BaXhwZFhlL2d2dHVLUDdraU10aWdzQ1dZRUxsblkzZ20wYndnTDlCTG5ybTFXCnZTS2hBb0dBWHZTRjB3cjFrV2VXRHdhcWZ4T2FyVGp2TUgyNEh4d2gzQUk0N3l0ZDhtV0V3Q3JXTWRSNnBuMVgKVkRjTHRxRXNJclFiL2RZVW9ITlVlRlowc2dLQjM0L3lXdlJXZTg3eGxZU0FxRk5UcU9RM1dnemNMYkozRjdpMgpIbU5hZUF4TU5aVUlRNWJubDdhSDJyVG90bFVGaXpPNHJ6YXVsdWJ1UEx0b3FHalBkSDQ9Ci0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0t
```
Comment (xh3b4sd) [inline kubeconfig_uat_zwm5s_60d5f81ac6c5fc15c236362b677b1a4c18f9b06e.yaml]: Why do we commit private keys here?

File: cmd/runtests/runner.go [hunk 3]
```diff
@@ -58,9 +61,58 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
...
+		params = clusters.NewGetClusterV5Params().WithClusterID(clusterOneID)
+		result, err := giantSwarmClient.Clusters.GetClusterV5(params, authWriter)
+		if err != nil {
+			cliutil.ExitIfError(microerror.Mask(err))
+		}
+
+		clusterOneAPIEndpoint = result.Payload.APIEndpoint
+	}
+
+	time.Sleep(3 * time.Second)
```
Comment (xh3b4sd) [inline cmd/runtests/runner.go:86]: Why is the delay here still necessary? After we improved `cluster-service` with a backoff for the key pair creation I thought this is fine now.

**Review by xh3b4sd — COMMENTED** [review]
I would appreciate to merge smaller PRs as this thing is surely getting out of hand. Can we please iterate in smaller steps? It is really not getting better to work or review with that way. At some point I am out of capacity to review these big PRs.

File: kubeconfig_uat_zwm5s_60d5f81ac6c5fc15c236362b677b1a4c18f9b06e.yaml (hunk 2 above)
Comment (marians) [reply kubeconfig_uat_zwm5s_60d5f81ac6c5fc15c236362b677b1a4c18f9b06e.yaml]: Accidentally. As both the key-pair and the clusters are short-lived, no danger.

File: cmd/runtests/runner.go (hunk 3 above)
Comment (marians) [reply cmd/runtests/runner.go:86]: No, actually cluster details couldn't be fetched right after cluster creation either.

File: helloworld-manifest.yaml (hunk 1 above)
Comment (marians) [reply helloworld-manifest.yaml]: Because I didn't know it. And now that I know the repo, my eagerness to use a completely undocumented repo is zero.

Comment (marians) [reply helloworld-manifest.yaml]: Anyway, I'll use it, as the option to have long-running requests with `/delay/5` will be helpful.

Comment (xh3b4sd) [reply helloworld-manifest.yaml]: Check the e2e tests in `aws-operator`. The draining tests use the e2e app.

**Comment by marians** [comment]
Merged, as this is heavily in-progress anyway.

**Pull request merged**